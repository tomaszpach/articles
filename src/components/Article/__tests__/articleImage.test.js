import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import ArticleImage from "../articleImage";

describe("ArticleImage", () => {
  test("should correctly render ArticleImage component", () => {
    const wrapper = shallow(<ArticleImage img="https://image.shutterstock.com/image-vector/default-ui-image-placeholder-wireframes-260nw-1037719192.jpg" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
