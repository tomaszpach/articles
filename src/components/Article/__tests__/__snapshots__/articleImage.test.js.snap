// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage should correctly render ArticleImage component 1`] = `
<img
  className="article-image"
  src="https://image.shutterstock.com/image-vector/default-ui-image-placeholder-wireframes-260nw-1037719192.jpg"
/>
`;
